fixtures/file

```diff
@@ -103,3 +103,5 @@ doi:10.1392/BC1.0
 10.1000/123
 
 a80122b2565c3e26a61cbf58d1d1aad7-us5
+
+RSSMRA00A01H501C
```

Comment:
Remove this. Fixture file is used for click tests and we do not want to slow down these.